python/img_hdf5_reader.py

```diff
@@ -58,12 +66,12 @@ def scale_sample(self, channel, sample, max_allowed, clip):
         [0.0, 127.0, 254.0, 254.0, 254.0]
 
         '''
-        sampled = np.clip(self.sample_image(channel, sample), 0, clip)
+        sampled = self.sample_image(channel, sample, use_dask).clip(0, clip)
         # 255 displays as black... color table issue?
         return sampled / clip * (max_allowed - 1)
 
     def to_png(self, channel, sample, png_path, clip):
-        MAX_ALLOWED = 256
+        MAX_ALLOWED = 65536
```

Comment:
Are we still making PNGs? Maybe get rid of this function? I don't know if 16-bit PNGs are a thing.